Python/bytecodes.c

```diff
@@ -3821,9 +3821,9 @@ dummy_func(
         }
 
         inst(CONVERT_VALUE, (value -- result)) {
-            convertion_func_ptr  conv_fn;
+            conversion_func conv_fn;
             assert(oparg >= FVC_STR && oparg <= FVC_ASCII);
-            conv_fn = CONVERSION_FUNCTIONS[oparg];
+            conv_fn = _PyEval_ConversionFuncs[oparg];
```

Comment:
We could make `inst(CONVERT_VALUE` `replicate(4) inst(CONVERT_VALUE` and rely on clang removing the lookup.

Reply:
Is there a way to do that where a fifth generic version isn't generated too? Otherwise, it doesn't help, so we might as well stay with this?